appveyor.yml

```diff
@@ -0,0 +1,25 @@
+init:
+  - git config --global core.autocrlf true
+
+branches:
+  only:
+    - appveyor
+
+before_build:
+  - '"%VS140COMNTOOLS%VsDevCmd.bat"'
+  - BeforeBuild.cmd
+
+build:
```

Comment:
Add the platform and configuration settings. 

platform:
  - Any CPU
configuration:
  - Debug
  - Release